crate_universe/src/utils/starlark/glob.rs

```diff
@@ -91,3 +94,49 @@ impl<'de> Visitor<'de> for GlobVisitor {
         })
     }
 }
+
+#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Deserialize)]
+#[serde(untagged)]
+pub enum GlobOrLabels {
```

Comment:
Can you comment on why this exists?

Reply:
It's already documented where it's used (the `TargetAttributes` declaration) but I'll add it here too.